src/main/resources/admin/tools/content-type-manager/content-type-manager.html

```diff
@@ -7,6 +7,19 @@
 
     <title>Content Type Manager</title>
 
+    <link rel="stylesheet" type="text/css" data-th-href="|${assetsUrl}/admin/common/styles/lib.css|">
```

Comment:
It seems that this lib.css file adds "overflow-y: hidden;" to the body, therefore the scroll bar is not properly working for big enough ( > view height) content types.

It also adds a blackish border in the buttons when they are clicked.

Reply:
We might need to use some `!important` styling in the app css 🙈